rllib/algorithms/ppo/tests/test_ppo_rl_module.py

```diff
@@ -48,30 +45,32 @@ def get_expected_model_config(
     obs_dim = env.observation_space.shape[0]
 
     if lstm:
-        encoder_config = LSTMEncoderConfig(
+        base_encoder_config = LSTMEncoderConfig(
             input_dim=obs_dim,
             hidden_dim=32,
             batch_first=True,
             num_layers=1,
             output_dim=32,
         )
     else:
-        encoder_config = MLPEncoderConfig(
+        base_encoder_config = MLPEncoderConfig(
             input_dim=obs_dim,
             hidden_layer_dims=[32],
-            hidden_layer_activation="ReLU",
+            hidden_layer_activation="relu",
             output_dim=32,
         )
 
-    pi_config = MLPConfig(
+    encoder_config = ActorCriticEncoderConfig(base_encoder_config=base_encoder_config)
+
+    pi_config = MLPModelConfig(
         input_dim=32,
         hidden_layer_dims=[32],
-        hidden_layer_activation="ReLU",
+        hidden_layer_activation="relu",
     )
-    vf_config = MLPConfig(
+    vf_config = MLPModelConfig(
         input_dim=32,
         hidden_layer_dims=[32],
-        hidden_layer_activation="ReLU",
+        hidden_layer_activation="relu",
```

Comment:
We "traditionally" write these in lower case in model_config and later convert them to something like ReLU in framework-specific code. This is now happening here as well.